docs/mining/overview.md

```diff
@@ -26,74 +26,141 @@ transactions waiting to be processed in the mempool.
 Decred uses the [BLAKE-256](../research/blake-256-hash-function.md) hashing function. PoW mining can be done using GPUs. However, ASICs that mine DCR are now available, and since their introduction the PoW difficulty has become high enough to make GPU
 mining unlikely to be profitable.
 
----
+## Purchasing specialized hardware (ASIC)
 
-## Solo Mining or Pool Mining  
+ASIC are the machines used to mine the most popular cryptocurrencies and can only be used for mining a single algorithm. 
 
-##### <img class="dcr-icon" src="/img/dcr-icons/Solo.svg" /> Solo Mining
+The following Decred ASIC miners are available on the market today:
```

Comment:
There are more miners available than just these three, for example:
* Obelisk DCR1
* Innosilicon D9
* Antminer DR3

In order to make this line a little more accurate, and to prevent the need of constantly maintaining an exhaustive list, lets update this line to something like 

> Here is a selection of the Decred ASIC miners which are available on the market today:

Reply:
Updated